### PR TITLE
Removed unused seldon analytics variables and updated docs

### LINKED ISSUE #1539: seldon-core-analytics should not use hostPort
hostPort for a Pod should not be specified unless it is absolutely necessary. When you bind a Pod to a hostPort, it limits the number of places the Pod can be scheduled, because each <hostIP, hostPort, protocol> combination must be unique. If you don’t specify the hostIP and protocol explicitly, Kubernetes will use 0.0.0.0 as the default hostIP and TCP as the default protocol.

### PR DESCRIPTION
**Pull request by axsaucedo**
* Fixes #1539
* Adds small heads up that all parameters from analytics charts can be overriden
* Merged the custom metrics page into the general prometheus

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **axsaucedo**

If they are not already assigned, you can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Thu Mar 26 20:17:45 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1620/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1620 --build=2

**Comment by seldondev** [comment]
Thu Mar 26 20:18:04 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1620/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1620 --build=1

File: doc/source/analytics/analytics.md [hunk 1]
```diff
@@ -48,8 +50,121 @@ kubectl port-forward svc/seldon-core-analytics-prometheus-seldon 3001:80 -n seld
...
+## Supported wrappers
+
+At present the following Seldon Core wrappers provide integrations with custom metrics:
+
+ * [Python Wrapper](../python/index.html)
 
 
 ## Example
 
-There is [an example notebook you can use to test the metrics](../examples/metrics.html).
```
Comment (RafalSkolasinski) [inline doc/source/analytics/analytics.md]: Maybe we should leave links to both examples?

Comment (RafalSkolasinski) [reply doc/source/analytics/analytics.md]: Ah, I see it above. My bad.

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/1620#pullrequestreview-382770637" title="Approved">RafalSkolasinski</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **axsaucedo**

If they are not already assigned, you can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/1620#issuecomment-604946477" title="Approved">RafalSkolasinski</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **axsaucedo**

If they are not already assigned, you can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/1620#issuecomment-605705465" title="Approved">axsaucedo</a>*, *<a href="https://github.com/SeldonIO/seldon-core/pull/1620#issuecomment-604946477" title="Approved">RafalSkolasinski</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Sun Mar 29 21:38:46 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1620/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1620 --build=4

**Comment by seldondev** [comment]
Sun Mar 29 21:38:49 UTC 2020
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1620/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1620 --build=3

**Pull request merged**